### ISSUE TITLE
Console.log causes the browser to get stuck

### ISSUE THREAD

**Issue by xrkffgg** [codesandbox/codesandbox-client#5656]
# 🐛 bug report

## Preflight Checklist

- [ ] I have read the
      [Contributing Guidelines](https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md)
      for this project.
- [ ] I agree to follow the
      [Code of Conduct](https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md)
      that this project adheres to.
- [ ] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

Console.log causes the browser to get stuck

### Link to sandbox: [link]() (optional)

https://codesandbox.io/s/8w24l?file=/index.js

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |  |
| Browser          |  Chrome |
| Operating System |  |

**Comment by andrasbacsai** [comment]
Hey,

Thanks for reporting. Very weird issue. We will take a look at it!

**Comment by CompuIves** [comment]
I believe the serializing of the content within `console-feed` goes wrong here. It might be that the object is huge? @samdenty maybe you know immediately what happens here, we could open a PR to fix this if you're interested!

**Comment by jueinin** [comment]
any update？

**Comment by lbogdan** [comment]
@jueinin Not really, I just opened an [issue in `console-feed`](https://github.com/samdenty/console-feed/issues/87) repository.
